src/pages/ArchivePage.js

```diff
@@ -0,0 +1,62 @@
+import styled from 'styled-components';
+import { useNavigate } from 'react-router-dom';
+import Header from '../components/Header.js';
+import Content from '../components/Content.js';
+import DiveWish from '../components/DiveWish.js';
+import IconButton from '../components/IconButton.js';
+import ArrowBack from '../images/ArrowBack.svg';
+import { useEffect } from 'react';
+
+export default function ArchivePage({
+  archivedWishes,
+  onToggleCheckmark,
+  onEditDiveWish,
+  onDeleteDiveWish,
+}) {
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    if (archivedWishes.length === 0) {
+      navigate('/');
+    }
+  }, [archivedWishes, navigate]);
```

Comment:
What is this code for? To directly get to the start page when deleted the last archived item?

Reply:
Exactly! As I only want the ArchivePage to be visible if there is already an archived item. Without that code, the user would have to go back manually once he/she takes off the last item of the archive page